socrates/lib/notifications/index.js

```diff
@@ -60,12 +59,37 @@ module.exports = {
       var filename = path.join(__dirname, 'jade/waitinglistConfirmation.jade');
       var receivers = [member.email()];
       notifications._sendMail(receivers, 'SoCraTes Waitinglist Confirmation', jade.renderFile(filename, options));
-      memberstore.allMembers(function (err, members) {
-        if (err || !members) { return logger.error(err); }
+      membersService.superuserEmails(function (err, superusers) {
+        if (err || !superusers) { return logger.error(err); }
         var filenameSuperuser = path.join(__dirname, 'jade/superuserWaitinglistNotification.jade');
-        var superusers = Member.superuserEmails(members);
         notifications._sendMail(superusers, 'New SoCraTes Waitinglist Entry', jade.renderFile(filenameSuperuser, options));
       });
     });
+  },
+
+  paymentConfirmation: function (memberID) {
```

Comment:
Ach so, das hier soll nicht für die eingetroffenen Überweisungen sein...

Sorry, der Text 

"Hello #{member.displayName()},<br>
+      we have received your payment. This makes you a participant." 

hat mich völlig in die Irre geführt. Ich persönlich würde das auf keinen Fall so schreiben, wenn wir das Geld noch nicht wirklich auf dem Konto haben! Ansonsten haben wir das Hinterherrennen...

Eher "thanks for notifying us that you've transferred the registration fee. bla bla" oder so.